docs/crd.md

```diff
@@ -0,0 +1,49 @@
+# Restoring Deleted CRs
+
+When a Rook CR is deleted, the Rook operator will respond to the deletion event to attempt to clean up the cluster resources. If any data is still present in the cluster, Rook will refuse to delete the CR to ensure data is not lost. The operator will refuse to remove the finalizer on the CR until the underlying data is deleted.
+
+While the underlying Ceph data and daemons continue to be available, the CRs will be stuck indefinitely in a Deleting state in which the operator will not continue to ensure cluster health. Upgrades will be blocked, further updates to the CRs are prevented, and so on. Since Kubernetes does not allow undeleting resources, the command below will allow repairing the CRs without even necessarily suffering cluster downtime.
+
+## Restore Command
+
+```bash
+kubectl rook-ceph restore-deleted <CRD> [CRName]
+
+# `restore-deleted`: is the command to restore the deleted CR
+# `<CRD>`: the CRD type that is to be restored, such as CephCluster, CephFilesystem, CephBlockPool and so on.
```

Comment:
Put these argument descriptions in bullets instead of inside the code block. The code block is generally just used for commands and their output. Also, I'd suggest moving these argument descriptions above the command example (before line 10)
```suggestion
- `<CRD>`: the CRD type that is to be restored, such as CephCluster, CephFilesystem, CephBlockPool and so on.
```